CharacterGenerator/CharacterGenerator/AllCharactersTableViewController.m

```diff
@@ -0,0 +1,154 @@
+//
+//  AllCharactersTableViewController.m
+//  CharacterGenerator
+//
+//  Created by Artur Lan on 8/2/15.
+//  Copyright (c) 2015 Mike Kavouras. All rights reserved.
+//
+
+#import "AllCharactersTableViewController.h"
+#import "characterModel.h"
+#import "Character.h"
+#import "characterDeetsViewController.h"
+#import "newCharacterTableViewController.h"
+
+@interface AllCharactersTableViewController ()
+@property (nonatomic) characterModel * model;
+
+@end
+
+@implementation AllCharactersTableViewController
+
+//Delete Action
+- (void)tableView:(UITableView *)tableView commitEditingStyle:(UITableViewCellEditingStyle)editingStyle forRowAtIndexPath:(NSIndexPath *)indexPath
+{
+    if (editingStyle == UITableViewCellEditingStyleDelete)
+    {
+        [self.model.allCharacters removeObjectAtIndex:indexPath.row];
+        // Delete the row from the data source
+        [tableView deleteRowsAtIndexPaths:@[indexPath] withRowAnimation:UITableViewRowAnimationAutomatic];
+    } 
+}
+
+// end Delete action
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    self.title = @"AC 2.2 Super Heroes";
+    
```

Comment:
Blank lines are good when they help organize your thoughts/group code in to logical chunks. This one is unnecessary.